src/components/Showcase/Showcase.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+/**
+ * NOTE: This component is meant to be used within React Storybook.
+ * It's a dev-only component, not meant to be used within Guppy.
+ */
```

Comment:
It feels weird to me to have this dev-only component live with all the others in `src/components` :/

I first thought that maybe we could rename it `StorybookShowcase`, but that felt like a really verbose name, which you need to type a lot in stories.

Maybe we could add a new directory for it + the wrapper component in `.storybook/config`?

Reply:
> yes, that feels a bit weird.
> But I would create a folder .storybook/components then it's clear that there are the storybook related conponents (wrapper + showcase).

Yeah, that's probably a good idea. The one reason I have against it is that it's a pain to import from a component (`../../.storybook/components/` instead of just `../`), but that's a really minor bit of friction. Most of the time you'll probably just copy/paste an existing story anyway.

Reply:
Maybe webpack could help here if we're adding a path resolve like `story-components/ShowCase` and webpack uses the `.storybook/components` location to import it.

But I think adding the relative path is OK and no big deal.